packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -56,7 +56,7 @@ export const enableSchedulingProfiler =
 // For now, we'll turn it on for everyone because it's *already* on for everyone in practice.
 // At least this will let us stop shipping <Profiler> implementation to all users.
 export const enableSchedulerDebugging = true;
-
+export const warnOnSubscriptionInsideStartTransition = false;
```

Comment:
Nit: Could you import this from ReactFeatureFlags.www-dynamic instead?

That way we get the test coverage but also the next time someone syncs to www they're less likely to accidentally turn this on